igor-web/igor-web.gradle

```diff
@@ -63,13 +63,3 @@ dependencies {
     testCompile 'com.github.tomakehurst:wiremock-jre8:2.22.0'
     testCompile 'org.mockito:mockito-core:2.6.8'
 }
-
-configurations.all {
-    resolutionStrategy {
-        force 'org.apache.log4j:log4j:1.2.17'
```

Comment:
There is no such group as `org.apache.log4j`. This was always a force that did nothing.

Reply:
maybe this was just future proofing? :grin: 

FWIW the igor boot2 branch has overhauled a bunch of the gradle madness to use a BOM and got rid of the resolutionStrategy as a result